src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/icons.fluent.scss

```diff
@@ -6452,3 +6452,11 @@
 .bit-icon--StoreLogoMed::before {
     content: '';
 }
+
+.bit-icon--ChevronDown::before {
+    content: '';
+}
+
+[dir='rtl'] .bit-icon--ChevronDown::before {
+    content: '';
+}
```

Comment:
what are these?
it's defined before and I don't think it is direction-dependant.